### PR TITLE
chore: add mitmproxy and mypy as dev dependendencies 

### PR DESCRIPTION
**Pull request by NiceAesth**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@ charset-normalizer==2.1.1 ; python_full_version >= '3.6.0'
 click==8.1.3 ; python_version >= '3.7'
 databases[mysql]==0.7.0
 datadog==0.44.0
-fastapi==0.88.0
+fastapi==0.89.0
```
Comment (cmyui) [inline requirements.txt:16]: this includes a breaking change for return type annotations on api endpoints, we should make sure no endpoints have any return annotations 

https://github.com/tiangolo/fastapi/releases/tag/0.89.0

Comment (NiceAesth) [reply requirements.txt:16]: checked everything that had the `@router` decorator and i can't seem to find any that have annotations

File: Pipfile [hunk 2]
```diff
@@ -30,6 +30,7 @@ pytest = "*"
 pre-commit = "*"
 black = "*"
 reorder-python-imports = "*"
+mitmproxy = "*"
```
Comment (cmyui) [inline Pipfile:33]: no mypy?

Comment (NiceAesth) [reply Pipfile:33]: huh?? i guess my changes got discarded, fixing

Comment (NiceAesth) [reply Pipfile:33]: fixed, looking like a squash and merge angle

**Review by cmyui — APPROVED** [review]

**Pull request merged**